pyvisa-py/gpib.py

```diff
@@ -65,10 +65,15 @@ def get_low_level_info(cls):
         return 'via Linux GPIB (%s)' % ver
 
     def after_parsing(self):
-        minor = self.parsed.board
-        pad = self.parsed.primary_address
-        self.handle = gpib.dev(int(minor), int(pad))
-        self.interface = Gpib(self.handle)
+        minor = int(self.parsed.board)
+        pad = int(self.parsed.primary_address)
+        sad = 0
+        timeout = 13
+        send_eoi = 1
+        eos_mode = 0
+        self.interface = Gpib(name = minor, pad = pad, sad = sad, timeout = timeout, send_eoi = send_eoi, eos_mode = eos_mode)
```

Comment:
I am trying to move toward PEP8 compliant code so could you remove the spaces around the = for keyword arguments.